gpii/node_modules/flowManager/configs/gpii.flowManager.config.base.json

```diff
@@ -0,0 +1,36 @@
+{
+    "type": "gpii.flowManager.config.base",
+    "options": {
+        "mainServerPort": 8081,
+        "distributeOptions": {
+            "flowManager.deviceReporterDataSource": {
+                "record": {
+                    "type": "kettle.dataSource.URL",
+                    "options": {
+                        "url": "http://localhost/device",
+                        "port": "{kettle.config}.options.mainServerPort"
+                    }
+                },
+                "target": "{that flowManager deviceReporterDataSource}"
+            }
+        },
+        "components": {
+            "server": {
+                "type": "kettle.server",
+                "options": {
+                    "gradeNames": "kettle.server.ws",
+                    "port": "{kettle.config}.options.mainServerPort",
+                    "components": {
+                        "flowManager": {
+                            "type": "gpii.flowManager",
+                            "options": {
+                                "gradeNames": ["gpii.flowManager.ws", "gpii.flowManager.save", "gpii.contextManager.app"]
```

Comment:
Why is contextManager located here as opposed to having a separate config like everything else? Is this just because it's something that will always live with the flowmanager?